src/components/AnimatedButton/AnimatedButton.tsx

```diff
@@ -63,6 +63,16 @@ const AnimatedButton: React.FC<Props> = ({
     }
     return ''
   }
+
+  const getContent = () => {
```

Comment:
move this to swap and set class through className prop; also template strings are not necessary there